structured/sql_api.proto

```diff
@@ -93,6 +93,6 @@ message SQLRequest {
 
 message SQLResponse {
   optional SQLResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
-  // The result set.
+  // The result set for the last Cmd in the request.
```

Comment:
Okay will add that

On Tue, Jun 23, 2015, 2:07 PM Peter Mattis <notifications@github.com> wrote:

> In structured/sql_api.proto
> <https://github.com/cockroachdb/cockroach/pull/1443#discussion_r33072126>:
>
> > @@ -93,6 +93,6 @@ message SQLRequest {
> >
> >  message SQLResponse {
> >    optional SQLResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
> > -  // The result set.
> > +  // The result set for the last Cmd in the request.
>
> Each result set definitely needs the list of columns returned. I totally
> missed that the list of columns returned was not part of the response.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1443/files#r33072126>.
>